test/src/tests-identity.ts

```diff
@@ -2386,32 +3171,22 @@ describe('identity', function() {
             result = resp;
         };
 
-        mockServer.respondWith(urls.identify, [
-            200,
-            {},
-            JSON.stringify({ mpid: 'MPID1', is_logged_in: false, status: 200 }),
-        ]);
+        fetchMockSuccess(urls.identify, {
+            mpid: 'MPID1',
+            is_logged_in: false,
+        });
 
+        // Init fires an identify call
         mParticle.init(apiKey, window.mParticle.config);
 
-        const identityRequest = { userIdentities: { customerid: 'test123' } };
-        function loginCallback(result) {
-            loginResult = result;
-        }
-        function logoutCallback(result) {
-            logoutResult = result;
-        }
-        function modifyCallback(result) {
-            modifyResult = result;
-        }
-
-        mParticle.Identity.login(identityRequest, loginCallback);
-        mParticle.Identity.logout(identityRequest, logoutCallback);
-        mParticle.Identity.modify(identityRequest, modifyCallback);
+        waitForCondition(() => {
+            return mParticle.Identity.getCurrentUser()?.getMPID() === 'MPID1';
+        })
+        .then(() => {
 
         result.should.have.properties('body', 'httpCode', 'getUser');
         result.httpCode.should.equal(200);
-        result.body.should.have.properties('mpid', 'status');
+        result.body.should.have.properties('mpid');
```

Comment:
was `status` removed intentionally?

Reply:
I don't know why it was passing in the first place. `status` isn't a valid IdentityResult.